### ISSUE TITLE
fix GitLab database model versioning

### ISSUE THREAD

**Issue by nazavode** [tracboat/tracboat#19]
As pointed out by @glensc , the model filename for GitLab ver. `9.0.0` should be `model90.py`. **This implies that GitLab doesn't change the database schema between patch releases (I don't think it is somehow guaranteed since the schema itself is not part of their public API)**.

It could be useful to make the separation between major and minor in model file name explicit, e.g. rename `model817.py` to `model8.17.py` or `model_8_17.py`, something like that, bare `817` is a bit ambiguous imho.

**Comment by glensc** [comment]
yes, `817` is ambiguous, use `8.17` or `8_17` if `.` (dot) not allowed in module name.

model changes in patch releases are not quaranteed, but it's highly unlikely to change.
